src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/Sdk/Eol.targets

```diff
@@ -28,7 +28,7 @@ Things to note:
   </PropertyGroup>
   <ItemGroup>
     <SdkSupportedTargetPlatformVersion Include="21.0" />
-    <EolWorkload Include="net6.0-android" Url="https://aka.ms/maui-support-policy" />
+    <EolWorkload Include="net$(TargetFrameworkVersion.TrimStart('vV'))-android" Url="https://aka.ms/maui-support-policy" />
```

Comment:
The weird expression here is from the .NET SDK:

https://github.com/dotnet/sdk/blob/208ccf0bffe0a09a2aec8587ce87bf9b357f161d/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.TargetFrameworkInference.targets#L59